shared/constants/chat.js

```diff
@@ -144,10 +141,14 @@ export type InboxState = Record<{
 
 export type MetaData = Record<{
   fullname: string,
+  brokenTracker: boolean,
```

Comment:
i was leaning towards having one source of truth and not having to merge that kind of info. We can discuss if you feel strongly about it. I did thinking about it but ended up leaving the followingMap (which is used elsewhere) as a standalone thing